proposals/introduce_ebpf.md

```diff
@@ -0,0 +1,82 @@
+# Introduce ebpf
+
+- **Authors**: Eileen Yu (@Eileen-Yu), Jim Bugwadia (jim@nirmata.com)
+- **Created**: Feb 15th, 2023
+- **Updated**:
+- **Abstract**: Introduce ebpf to trigger kyverno policy for extended use cases
+
+## Contents
+
+- [Introduction](#introduction)
+- [The Problem](#the-problem)
+- [Proposed Solution](#proposed-solution)
+- [Open Question](#open-question)
+
+## Introduction
+
+Kyverno is now running as a dynamic admission controller for policy management. This allows users to manage Kubernetes native resources.
+
+This proposal aims to extend the ability of kyverno by introduing ebpf, so that it could cover events at a finer granularity at runtime.
+
+## The Problem
+
+At current stage kyverno focus on enforcing policies based on admission webhook. This enable users to define policy using k8s manifests to easily manage resources.
+
+Well, it is common for users to raise further fine-grained requirements out of security concern. This goes far beyond simple "approve / deny" the request.
+One instance is, users may want to limit the volume mount to a specific container. Another example is to control the access to certain API paths based on layer 7 calls.
+
+It would be better if there is a way to enforce these granular configurations as K8s policies without needing to learn new language.
+
+## Proposed Solution
```

Comment:
I know this is (currently) marked as a WIP, but the proposed solution here I don't quite follow. It sounds like you're proposing something like

```mermaid
flowchart LR
    id1[eBPF Tool] --> id2[K8s Event] --> id3[Kyverno validate rule] --> id4[Action]
```